src/Models/IoTContainerizedProject.ts

```diff
@@ -149,7 +153,10 @@ export class IoTContainerizedProject extends IoTWorkbenchProjectBase {
           `Internal Error. Could not find iot workbench project file.`);
     }
 
-    // Step 3: Open project
+
+    // Step 3: Configure project

```

Comment:
Please mark TODO if it is not implemented.

Reply:
Fixed.